### PR TITLE
Move key batches handler

### PR DESCRIPTION
**Pull request by brinxmat**

### CODE REVIEW

File: bulk-export/src/main/java/no/sikt/nva/data/report/api/export/CsvTransformer.java [hunk 1]
```diff
@@ -63,6 +63,11 @@ protected void persist(List<ContentWithLocation> transformedData) {
         transformedData.forEach(this::persist);
     }
 
+    private void persist(ContentWithLocation transformation) {
```
Comment (brinxmat) [inline bulk-export/src/main/java/no/sikt/nva/data/report/api/export/CsvTransformer.java:66]: Overload methods
Seek the company of friends
Checkstyle satisfied

File: template.yaml [hunk 2]
```diff
@@ -239,23 +239,13 @@ Resources:
             Type: SQS
             Destination: !GetAtt TransformPersistedResourceDLQ.Arn
 
-
-  ResetDatabaseButton:
```
Comment (brinxmat) [inline template.yaml:243]: When a lambda leaves
Sometimes a vague trace remains
As yesteryear's snows

**Review by monaullah — APPROVED** [review]
Very much appreciated and approved

File: bulk-export/src/test/java/no/sikt/nva/data/report/api/export/GenerateKeyBatchesHandlerTest.java [hunk 3]
```diff
@@ -174,7 +175,7 @@ private KeyBatchRequestEvent saveContainedEvent(PutEventsRequest putEventsReques
             PutEventsRequestEntry eventEntry = putEventsRequest.entries()
                                                    .stream()
                                                    .collect(SingletonCollector.collect());
-            return attempt(eventEntry::detail).map(
+            return Try.attempt(eventEntry::detail).map(
```
Comment (monaullah) [inline bulk-export/src/test/java/no/sikt/nva/data/report/api/export/GenerateKeyBatchesHandlerTest.java:178]: Why try?

Comment (brinxmat) [reply bulk-export/src/test/java/no/sikt/nva/data/report/api/export/GenerateKeyBatchesHandlerTest.java:178]: Some say one just must —
Ambition is critical!
Here, though, JetBrains helped.

**Pull request merged**